frontend/www/js/omegaup/components/problem/Collection.vue

```diff
@@ -53,34 +59,54 @@
       <div class="card-body panel-body">
         <div class="container-fluid">
           <div class="row d-flex justify-content-center">
-            <omegaup-problem-collection
-              :href="'/problem/author/'"
-              :title="T.problemCollectionAuthors"
-            >
+            <omegaup-problem-collection :title="T.problemCollectionAuthors">
               <template #icon>
                 <font-awesome-icon :icon="['fas', 'user']"></font-awesome-icon>
               </template>
+              <template #collection-button>
+                <a class="btn btn-primary" href="/problem/author/">{{
+                  T.problemcollectionViewProblems
+                }}</a>
+              </template>
             </omegaup-problem-collection>
             <omegaup-problem-collection
-              :href="'/problem/random/'"
               :title="T.problemCollectionRandomProblem"
             >
               <template #icon>
                 <font-awesome-icon
                   :icon="['fas', 'random']"
                 ></font-awesome-icon>
               </template>
+              <template #collection-button>
+                <a class="btn btn-primary" href="/problem/random/">{{
+                  T.problemcollectionViewProblems
+                }}</a>
+              </template>
             </omegaup-problem-collection>
             <omegaup-problem-collection
-              :href="'/problem/'"
               :title="T.problemCollectionSearchProblem"
             >
               <template #icon>
                 <font-awesome-icon
                   :icon="['fas', 'search']"
                 ></font-awesome-icon>
               </template>
+              <template #collection-button>
+                <a
+                  class="btn btn-primary"
+                  href="#"
+                  @click="showFinderWizard = true"
+                  >{{ T.problemcollectionViewProblems }}</a
+                >
+              </template>
             </omegaup-problem-collection>
+            <!-- TODO: Migrar el problem finder a BS4 (solo para eliminar algunos estilos) -->
+            <omegaup-problem-finder
+              v-show="showFinderWizard"
+              :possible-tags="wizardTags"
+              @close="showFinderWizard = false"
+              @search-problems="wizardSearch"
```

Comment:
```suggestion
              @search-problems="$emit('search-problems', $event)"
```

esto funciona? si sí, no hay necesidad de tener `wizardSearch`! y también, cuando se está re-emitiendo un evento, hay que intentar usar el mismo nombre para que sea más rápido de encontrar las implementaciones.